classes/page_group.php

```diff
@@ -54,6 +83,27 @@ protected function set_fuzzydurationcounts(array $data) {
         $this->raw_set('fuzzydurationcounts', json_encode($data, JSON_FORCE_OBJECT));
     }
 
+    /**
+     * Called after the page group is created in the DB.
+     */
+    public function after_create() {
+        $this->after_update(true);
```

Comment:
This feels like an antipattern. Could we instead call the action we're expecting it to perform as part of creation?

For example:
```
$this->update_cache();
```